src/Cake.Common/Tools/OpenCover/OpenCoverSettings.cs

```diff
@@ -64,6 +69,7 @@ public OpenCoverSettings()
             _filters = new HashSet<string>(StringComparer.OrdinalIgnoreCase);
             _excludedAttributeFilters = new HashSet<string>(StringComparer.OrdinalIgnoreCase);
             _excludedFileFilters = new HashSet<string>(StringComparer.Ordinal);
+            SkipAutoProps = false;
```

Comment:
@erik-am since this is the default value for a object of type bool, I don't think that this is required.  If you can remove this, I will get this merged.